src/System.Private.DataContractSerialization/src/System.Private.DataContractSerialization.csproj

```diff
@@ -48,7 +47,7 @@
     <Compile Include="$(RuntimeSerializationSources)\ExtensionDataObject.cs" Condition="$(NetNative)" />
     <Compile Include="$(RuntimeSerializationSources)\ExtensionDataReader.cs" Condition="$(NetNative)" />
     <Compile Include="$(RuntimeSerializationSources)\Globals.cs" />
-    <Compile Include="$(RuntimeSerializationSources)\GenericParameterDataContract.cs" Condition="$(NetNative) Or $(MergeDcjs)" />
```

Comment:
We're using MERGE_DCJS directive so the $MergeDcjs is true. Therefore this condition is always true $(NetNative) Or $(MergeDcjs) so I've removed the condition altogether.